includes/payment-methods/class-afterpay-payment-method.php

```diff
@@ -32,29 +32,42 @@ public function __construct( $token_service ) {
 		$this->accept_only_domestic_payment = true;
 		$this->limits_per_currency          = [
 			'AUD' => [
-				'min' => 100,
-				'max' => 200000,
-			], // Represents AUD 1 - 2,000 AUD.
+				'AU' => [
+					'min' => 100,
+					'max' => 200000,
+				], // Represents AUD 1 - 2,000 AUD.
+			],
 			'CAD' => [
-				'min' => 100,
-				'max' => 200000,
-			], // Represents CAD 1 - 2,000 CAD.
+				'CA' => [
+					'min' => 100,
+					'max' => 200000,
+				], // Represents CAD 1 - 2,000 CAD.
+			],
 			'NZD' => [
-				'min' => 100,
-				'max' => 200000,
-			], // Represents NZD 1 - 2,000 NZD.
+				'NZ' => [
+					'min' => 100,
+					'max' => 200000,
+				], // Represents NZD 1 - 2,000 NZD.
+			],
 			'GBP' => [
-				'min' => 100,
-				'max' => 120000,
-			], // Represents GBP 1 - 1,200 GBP.
+				'UK' => [
+					'min' => 100,
+					'max' => 120000,
+				], // Represents GBP 1 - 1,200 GBP.
+			],
 			'USD' => [
-				'min' => 100,
-				'max' => 400000,
-			], // Represents USD 1 - 4,000 USD.
+				'US' => [
+					'min' => 100,
+					'max' => 400000,
+				], // Represents USD 1 - 4,000 USD.
+			],
 			'EUR' => [
-				'min' => 100,
-				'max' => 100000,
-			], // Represents EUR 1 - 1,000 EUR.
+				// TODO: Stripe docs https://stripe.com/docs/payments/afterpay-clearpay doesn't have EUR as supported currency.
+				'default' => [
```

Comment:
I guess to be updated yet?

Reply:
Yes, but I'm confused as to why we even have EUR listed as supported if Stripe docs doesn't have it